### PR TITLE
Bump dependencies on go modules

### PR DESCRIPTION
**Pull request by SeanCondon**
they had gone really out of date

### CODE REVIEW

**Comment by adibrastegarnia** [comment]
@SeanCondon 
This line needs to be changed because in the new version of onos-lib-go, I modified NB server to include security config information. 
https://github.com/onosproject/onos-topo/blob/master/cmd/onos-topo/onos-topo.go#L68

New function is here: 
https://github.com/onosproject/onos-lib-go/blob/master/pkg/northbound/server.go#L73

**Comment by SeanCondon** [comment]
@adibrastegarnia I think this is exactly the reason we should never be changing the signature of a method on `onos-lib-go` - instead the old method should be marked `deprecated` and a new method made with the new signature.

It's OK to change the implementation but not the interface

**Comment by adibrastegarnia** [comment]
@SeanCondon 
I agree with that and I wanted to avoid it using a builder approach (instead of sending lots of parameters to a function) for that function to avoid deprecating the old ones in case we want to add something into the config  (I was going to change it but stopped doing that to make changes less for now :)) and mentioned in my PR about it  to make sure all of the subsystems use the previous release :)

File: go.mod [hunk 1]
```diff
@@ -3,11 +3,12 @@ module github.com/onosproject/onos-topo
...
 
 require (
-	github.com/apex/log v1.3.0 // indirect
-	github.com/atomix/go-client v0.2.0
+	github.com/atomix/go-client v0.2.1
 	github.com/gogo/protobuf v1.3.1
+	github.com/mattn/go-colorable v0.1.2 // indirect
 	github.com/onosproject/helmit v0.6.4
-	github.com/onosproject/onos-lib-go v0.6.4
+	github.com/onosproject/onos-lib-go v0.6.7
```
Comment (adibrastegarnia) [inline go.mod:10]: after we merge this PR (https://github.com/onosproject/onos-lib-go/pull/51), we should bump to v0.6.8

**Pull request merged**